src/ZoneServer/World/Actors/Characters/Character.cs

```diff
@@ -1131,6 +1138,13 @@ public bool TakeDamage(float damage, ICombatEntity attacker)
 				return true;
 			}
 
+			// inform the follower that I got hit so it can aggro
+			if (this.Variables.Temp.TryGet<Mob>("Melia.BlueOrbSummon.Monster", out var follower))
```

Comment:
If we're going to extend the hit information, shouldn't we perhaps directly go for a more general approach? We might also have a need for monsters reacting to other monsters getting hit for example, and approaching this from this direction, where we inform all relevant parties that something happened, feels a little disjointed and tedious.

How about we let the AIs observe the world instead? If we were to add a hit server event we could potentially use it to check for all kinds of combat actions, and all the code would be in the AiScript, keeping things tidy.

Reply:
So are you suggesting that perhaps, all entities within an area are informed about hit events to any target?  This actually could be needed, for example in RO, there are the concept of Assist monsters, which aggro if a monster of the same species is hit within a certain range.  This could be used to make the pet aggro, too, eliminating the need for the player to inform them of the hit.  I feel like this is beyond the scope of this PR, though.